agent/containers/images/change-perm

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+AUTH="${1}"
+
+function change {
+    echo https://quay.io/api/v1/repository/pbench/$1/permissions/team/core
+    curl \
+        -X PUT \
+        -H "Content-Type: application/json" \
+        -H "Authorization: Bearer $AUTH" \
+        -d '{ "role": "admin" }' \
+        https://quay.io/api/v1/repository/pbench/$1/permissions/team/core
+}
+
+for i in workloads tools tool-meister tool-data-sink base all; do
+    change "pbench-agent-${i}-fedora-33"
+    change "pbench-agent-${i}-fedora-34"
+    change "pbench-agent-${i}-centos-7"
+    change "pbench-agent-${i}-centos-8"
+done
+
+exit 0
+
+change pbench-devel
```

Comment:
It would be good to include a comment explaining why there is code _after_ the `exit`.  (And, it would be good to be specific where we can, such as the `pbenchinacan*` images are private, because the server image is based on RHEL which we are not allowed to redistribute.)